composer.json

```diff
@@ -70,7 +70,8 @@
         "twig/twig": "3.3.8",
         "vlucas/phpdotenv": "5.4.1",
         "waryway/php-traits-library": "1.0.4",
-        "yubico/u2flib-server": "1.0.2"
+        "yubico/u2flib-server": "1.0.2",
+        "milo/schematron": "^1.0"
```

Comment:
should pin this to 1.0.1 (for security reasons, we are pinning all third party dependencies)
```
"milo/schematron": "1.0.1"
```

Reply:
this is a very old package, so will :crossed_fingers: that no php8 issues :)

Reply:
yep forgot! my mantra these days!